src/Calendar/Navigation.jsx

```diff
@@ -135,7 +136,9 @@ export default function Navigation({
         type="button"
       >
         {navigationLabel
-          ? navigationLabel({ date, view, label })
+          ? navigationLabel({
+            locale: locale || getUserLocale(), date, view, label,
```

Comment:
```suggestion
            locale: locale || getUserLocale(), 
            date, 
            view, 
            label,
```

Just for parity with 3.x branch :)